### DISCUSSION TITLE
View Page Source included JSON data?

### DISCUSSION THREAD

**Discussion by Uranbold** [vercel/next.js#17693] [Help] [answered]
Hello,

In the footer area. My website has included the JSON data with the following example.

`<script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"originalUrl":"","active_sub":"","gt_language":null,"post_type":"page","parent_1":"","parent_2 ....`

Can we hide this? Or this is necessary to hide? 

Used GetStaticProps.

Thanks.

**Comment by jamesmosier** [comment, marked as answer]
This is needed so Next knows what data to render while building your page on the server. It should not contain anything sensitive because you are using that same data to render your page 😄  There is no way to hide it.

**Reply by Uranbold** [reply]
Thanks for the answer :)

**Reply by mehmetakifakkus** [reply]
This JSON is included on the already built page. I mean already built html page is just about the JSON code. I think there is no need to build it again, then there is no need having the data. Am I missing sth?

**Comment by Rahulk4128** [comment]
Does this JSON data increase the size ? how can we trigger the size problems?
